.env.dev

```
@@ -52,7 +52,7 @@ PINECONE_TOP_K=5
 # AI/LLM SERVICES
 # =============================================================================
 HUGGINGFACE_HUB_TOKEN=your-huggingface-token
-HUGGINGFACE_CHAT_MODEL=microsoft/DialoGPT-medium
+HUGGINGFACE_CHAT_MODEL=bert-base-uncased
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HUGGINGFACE_CHAT_MODEL key should go before the HUGGINGFACE_HUB_TOKEN key

(UnorderedKey)